### ISSUE TITLE
Third-party-cookies console error

### ISSUE THREAD

**Issue by T34-active** [mdn/content#33574]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Privacy/Third-party_cookies

### What information was incorrect, unhelpful, or incomplete?

![image](https://github.com/mdn/content/assets/71604450/12428e02-5917-4c7d-b788-5d37e434787b)
![image](https://github.com/mdn/content/assets/71604450/0330555d-9d28-463b-902c-3c49c45e0e72)

### What did you expect to see?

The local directory has that image, see if you need to replace it?

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/privacy/third-party_cookies`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Privacy/Third-party_cookies
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/privacy/third-party_cookies/index.md
* Last commit: https://github.com/mdn/content/commit/4d26c993afa39460cafd2e4f87a58e19d5262a39
* Document last modified: 2024-05-06T08:50:52.000Z

</details>

**Comment by bsmth** [comment]
Good catch, this will be fixed long-term by https://github.com/mdn/yari/pull/10858 but we can revert to the local version until then

**Comment by bsmth** [comment]
The Yari release is merged, so this should be fixed in the nightly build when mdn/content brings in the latest Yari version.
